### ISSUE TITLE
Wizard fails too configure the integration, HA restarts

### ISSUE THREAD

**Issue by murlex** [Tasshack/dreame-vacuum#122]
**Describe the bug**
HA is rebooted on Submit button in configuration wizard. No integration is added upon restart.

**To Reproduce**
Add the integration with HACS, fill all wizard fields - looks ok, click Submit button. HA is restarted in few seconds.

**Expected behavior**
Wizard should add the integration successfully

**Screenshots**

**Additional Information (please complete the following information)**
 - Model Name: Dreame Z10 Pro
 - Firmware Version
 - Home Assistant Version: 2023.3.6
 - Configuration Type: With  map support
 - Errors or warnings shown in the HA logs (if applicable):

**Comment by oOJoshOo** [comment]
On my Homeassistent it now also fails to Configure

**Comment by Funkelfetisch** [comment]
yes I was out of memory, increasing VM mem from 2 to 4gb fixed it

**Comment by Renalds-B** [comment]
> I have added a new option in the entity configuration for rendering the map half size and it lowers memory consumption enough to make the integration work on an 1GB RPI3. This option also disables some map objects like mop path for saving extra ram.
> 
> https://github.com/Tasshack/dreame-vacuum/releases/tag/v2.0.0b6

Is this option still available? Can't find it. 
Also get crash on Pi3 1GB

**Comment by Tasshack** [comment]
It is only available on Beta.
